### DISCUSSION TITLE
Как заставить работать ChatGPT

### DISCUSSION THREAD

**Discussion by Lina28011** [Flowseal/zapret-discord-youtube#1056] [Q&A] [answered]
Можно ли сделать чтобы https://chatgpt.com открывался? понимаю что тут блок идет от них а не от нас, может можно тоже чтото придумать для таких ситуаций?

**Comment by serpen7** [comment]
если этот сайт сам ограничил регион своего функционирования то гудбайдпи и запрет вам не помогут, только впны. Которые показывают сайту что вы не с заблоченного региона.

**Comment by Eptiy** [comment, marked as answer]
Тут возможное решение описал: https://github.com/bol-van/zapret-win-bundle/issues/17#issuecomment-2559469940

Ещё для геоблокировок можно использовать смарт днс, например comss.one: https://www.comss.ru/page.php?id=7315

**Reply by Odinzov** [reply]
По верхней ссылке ошибка 404, можешь переписать пожалуйста?
![image](https://github.com/user-attachments/assets/96c5ad30-dbf6-423e-b935-0e79b35288f1)

**Reply by Eptiy** [reply]
https://brokencore.club/threads/44402/

**Reply by languagewow** [reply]
> https://brokencore.club/threads/44402/

Привет! По данному методу начинает работать чат на компьютере, но на айфоне при попытке авторизоваться вылетает ошибка по региону. Почему так?

**Reply by Eptiy** [reply]
А как ты на айфоне умудрился модифицировать hosts? Я с их железками не очень знаком, но точно могу сказать, что там такие фокусы не прокатят.

**Comment by G3-SCORPIO** [comment]
Поможет реальный VPN, а не мусор который продают в России - https://www.vpngate.net/en/download.aspx
